spyder/plugins/editor/plugin.py

```diff
@@ -209,10 +211,12 @@ def setup_page(self):
                                    'codecompletion/enter_key')
             calltips_box = newcb(_("Display balloon tips"), 'calltips')
             gotodef_box = newcb(_("Link to object definition"),
-                  'go_to_definition',
-                  tip=_("If this option is enabled, clicking on an object\n"
-                        "name (left-click + Ctrl key) will go this object\n"
-                        "definition (if resolved)."))
+                                'go_to_definition',
+                                tip=_("If this option is enabled, clicking on "
+                                      "an object\n"
+                                      "name (left-click + Ctrl key) will go "
+                                      "this object\n"

```

Comment:
-> "go *to* this object definition" (typo is present in the original)

Reply:
These errors are much easier to make when text strings are so broken up like this, FYI.